From cc3dcfac21349b8422ce258203fcb56dea257c27 Mon Sep 17 00:00:00 2001
From: Max-Julian Pogner <max-julian@pogner.at>
Date: Thu, 13 Jul 2017 15:27:32 +0200
Subject: [PATCH] cifs-utils: man mount.cifs: "support" written in catalan

The word "suports" is missing a second 'p'

catalan ref: https://en.wiktionary.org/wiki/suport
english ref: https://en.wiktionary.org/wiki/support
---
 mount.cifs.8 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mount.cifs.8 b/mount.cifs.8
index 01579f6..8c1ca37 100644
--- a/mount.cifs.8
+++ b/mount.cifs.8
@@ -324,7 +324,7 @@ See section \fIACCESSING FILES WITH BACKUP INTENT\fR for more details
 .PP
 nocase
 .RS 4
-Request case insensitive path name matching (case sensitive is the default if the server suports it)\&.
+Request case insensitive path name matching (case sensitive is the default if the server supports it)\&.
 .RE
 .PP
 ignorecase
-- 
2.1.4

